cmd/commands/integrations.go

```diff
@@ -41,9 +41,9 @@ type (
 )
 
 var gitProvidersByName = map[string]model.GitProviders{
-	"bitbucket-server": model.GitProvidersBitbucketServer,
-	"github":           model.GitProvidersGithub,
-	"gitlab":           model.GitProvidersGitlab,
+	"BITBUCKET-SERVER": model.GitProvidersBitbucketServer,
+	"GITHUB":           model.GitProvidersGithub,
+	"GITLAB":           model.GitProvidersGitlab,
```

Comment:
we can't change these.
that breaks the current cf api, and also it does not match the provider strings used by autopilot.